test/e2e/cloud-provider-azure.go

```diff
@@ -44,8 +45,19 @@ func InstallCalicoAndCloudProviderAzureHelmChart(ctx context.Context, input clus
 	specName := "cloud-provider-azure-install"
 	By("Installing cloud-provider-azure components via helm")
 	options := &helmVals.Options{
-		Values: []string{fmt.Sprintf("infra.clusterName=%s", input.ConfigCluster.ClusterName), fmt.Sprintf("cloudControllerManager.clusterCIDR=%s", strings.Join(cidrBlocks, `,`))},
+		Values:       []string{fmt.Sprintf("infra.clusterName=%s", input.ConfigCluster.ClusterName)},
+		StringValues: []string{fmt.Sprintf("cloudControllerManager.clusterCIDR=%s", strings.Join(cidrBlocks, `\,`))},
 	}
+	// If testing a CI version of Kubernetes, use CCM and CNM images built from source.
+	if useCIArtifacts {
```

Comment:
I assume we don't have coverage of this scenario in this PR CI, is there a test-infra test we can keep an eye on going forward after we merge this?

Reply:
we do actually, via 
pull-cluster-api-provider-azure-conformance-with-ci-artifacts and pull-cluster-api-provider-azure-windows-containerd-upstream-with-ci-artifacts